packages/libs/react-ui/.eslintrc.js

```diff
@@ -6,6 +6,15 @@ module.exports = {
     '@kadena-dev/eslint-config/profile/react',
     'plugin:storybook/recommended',
   ],
+  rules: {
+    '@typescript-eslint/consistent-type-imports': [
+      'error',
+      {
+        fixStyle: 'inline-type-imports',
+        prefer: 'type-imports',
+      },
+    ],
+  },
```

Comment:
Should we move this to the react profile instead?

Reply:
Yeah I'm in favour! Figured to at least implement it into our domain but I think it would be good to implement it as a rule everywhere.

Reply:
Done!